k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -157,8 +158,8 @@
 
     private static final Set<Flag> SYNC_FLAGS = EnumSet.of(Flag.SEEN, Flag.FLAGGED, Flag.ANSWERED, Flag.FORWARDED);
 
-    private void suppressMessages(Account account, List<LocalMessage> messages) {
-        EmailProviderCache cache = EmailProviderCache.getCache(account.getUuid(), context);
+    private void suppressMessages(String accountUuid, List<LocalMessage> messages) {
```

Comment:
You are quite right. I did this in an experimental inbetween version where calling sites were working with MessageReferences and didn't have the Account reference themselves, but later changed that back and forgot this bit.